src/ls/driver.ts

```diff
@@ -17,7 +17,7 @@ export default class SingleStoreDB<O = any> extends AbstractDriver<any, O> imple
       return this.connection;
     }
 
-    const pool = MySQLLib.createPool(this.credentials.connectString || {
```

Comment:
@AdalbertMemSQL as far as I understand, this code path is never triggered in SQL Tools - I don't see a way to supply a connection string, only (host,port,database,user,password).

Reply:
Yea, it looks like I just accidentally copied it from another driver.